src/pineko/check.py

```diff
@@ -41,3 +42,30 @@ def check_grid_and_eko_compatible(pineappl_grid, operators):
     # x-grid
     if not np.all(in1d(np.unique(operators["targetgrid"]), np.array(x_grid))):
         raise ValueError("x grid in pineappl grid and eko operator are NOT compatible!")
+
+
+def check_grid_contains_sv(grid_path, xir, xif, ftr):
+    """Raises a `ValueError if the theory_card asks for scale-variations but they are not
```

Comment:
I wonder whether we should split into two functions: one for ren and one for fact - since they are required in different situations